tests/test_data/test_transform.py

```diff
@@ -79,6 +80,45 @@ def test_resize():
     assert results['img_shape'] == (800, 1280, 3)
 
 
+def test_mosaic():
+    pipeline = [
+        dict(
+            type='Mosaic',
+            load_pipeline=[
+                dict(type='LoadImageFromFile', to_float32=True),
+                dict(type='LoadAnnotations', with_bbox=True)
+            ],
+            sub_pipeline=[
+                dict(
+                    type='PhotoMetricDistortion',
+                    brightness_delta=32,
+                    contrast_range=(0.5, 1.5),
+                    saturation_range=(0.5, 1.5),
+                    hue_delta=18),
+                dict(type='RandomFlip', flip_ratio=0.5)
+            ],
+            size=(640, 640),
+            jitter=0.2,
+            min_offset=0.2,
+            letter_box=False)
+    ]
+    # mosaic_module = build_from_cfg(transform, PIPELINES)
+    data_root = 'data/coco/'
```

Comment:
Normally unittest should not directly use a real dataset, since it would easily take more resources than the ci server can afford. Use dummy data instead.